javascript/implementation/CommandLineInterace.ts

```diff
@@ -92,19 +103,19 @@ export class CommandLineInterface {
             globalThis.database = this.instance;
             globalThis.root = this.instance.getGlobalDirectory();
             this.instance.addListener(
-                async (commitInfo: BundleInfo) => logToStdErr(`received commit: ${JSON.stringify(commitInfo)}`))
+                async (commitInfo: BundleInfo) => logToStdErr(`received commit: ${JSON.stringify(commitInfo)}`));
             for (const target of this.targets) {
                 logToStdErr(`connecting to: ${target}`);
                 try {
-                    await this.instance.connectTo(target, logToStdErr);
+                    await this.instance.connectTo(target, { onClose: logToStdErr, authToken: process.env["GINK_AUTH_TOKEN"] });
```

Comment:
This is assuming all targets will use the same auth key. Can you think of a better way of doing this?

Reply:
That's okay.  Though I think it might make more sense to use `GINK_AUTH` rather than `GINK_AUTH_KEY`

Reply:
or `GINK_TOKEN` (and then automatically prepending the `token<space>` part